src/emulator/util/src/util.cpp

```diff
@@ -79,6 +79,13 @@ void set_level(spdlog::level::level_enum log_level) {
     spdlog::set_level(log_level);
 }
 
+void add_sink(std::wstring log_path) {
+    sinks.push_back(std::make_shared<spdlog::sinks::basic_file_sink_mt>(log_path, true));
+
+    spdlog::set_default_logger(std::make_shared<spdlog::logger>("vita3k logger", begin(sinks), end(sinks)));
+    spdlog::set_pattern("%^[%H:%M:%S.%e] |%L| [%!]: %v%$");
```

Comment:
is this needed?

Reply:
Yes, both lines are needed, if the first is removed the log gets created but nothing is written to it, and if the second is removed the log will be unformatted

Reply:
ok, then making another function `logging::set_pattern` that just does this, and use it in init too, or make the pattern string a `static const char* LOG_PATTERN` and use it in init too.